user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
-# Welcome to the Strapi user documentation!
+# Welcome to the Strapi user guide!
 
-**Strapi is the open-source [Headless CMS](https://strapi.io) developers love.**
\ No newline at end of file
+This user guide contains the functional documentation related to all features available in the admin panel of your Strapi application. 
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development environment or Production** <br> When starting working on your application, it is in development environment, which is the status for the structuration and configuration phrase of your application and its content. After deploying your application, it is in production. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest version. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content created with Strapi itself, are ruled by a system of permissions. From your Strapi application, you have the possibility to define, at a detailed level, the roles and permissions of all admin panel users and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
```

Comment:
```diff
- the content created with Strapi itself
+ the content managed with Strapi itself
```

I don't know if `managed` is more global than `created` which imply just the "creation".
That's a question.

Reply:
```diff
- From your Strapi application
+ From your Strapi admin panel
```

Here is a global question, when should we use "Strapi application" and what is represented by that VS "Strapi admin panel"
I think it could be good to define that.

Reply:
```diff
- all admin panel users
+ all administrators (admin panel users)
```

I'm doing this suggestion to trigger a wording clarification to that specific point.
In this [Blog article](https://strapi.io/blog/why-we-split-the-management-of-the-admin-users-and-end-users) we are using `Admin users` and `End users` terms.
And here in the interface, it's different again.
![Screen Shot 2020-12-01 at 11 15 13](https://user-images.githubusercontent.com/5716596/100727590-be0d0e80-33c6-11eb-8201-fbef5bf5ee75.png)
I think a clarification should be done with @Aurelsicoko and @maevalienard I think.

Reply:
More general question, do we have a clarification as to what the actual term for the admin panel is?

Since the documentation is quite formal, is it:
- admin panel
- administration panel
- admin ui
- ect

And related to that what do we call those users?
- admin panel users
- administration panel users
- admin users
- ect

Reply:
I discussed this topic with Aurélien & Maeva. We have agreed that we should use "administrators" to talk about the users who have access to the admin panel (instead of "admin panel users" which sounds more complicated), and "end-users" for the users that use the actual result of what administrators create.
As for "admin panel", I also talked about it with the Product team a while ago and it seems like it's okay for everyone.